install/charts/templates/rbac.yaml

```diff
@@ -161,12 +161,15 @@ rules:
 - apiGroups: [""]
   resources: ["configmaps"]
   verbs: ["get", "create", "delete"]
+- apiGroups: ["apiextensions.k8s.io"]
```

Comment:
⚠️ This is something we definitely should NOT do! ⚠️
The idea behind using a separate ServiceAccount for karydia is to confine the permissions to only the ones which are really needed by karydia to run correctly. If we now add additional permissions just for the (helm) deletion step it is not security-aware.
Either we should create a separate ServiceAccount just for karydia installation / deletion with the respective (helm) `pre-` hook and directly delete it again after the operation OR we just use the `tiller` ServiceAccount which is there anyway and which has already the needed permissions (not sure how this works with "tillerless" Helm but it should be OK for now).

Reply:
As for now, we mixed all the permissions already. I believe, we should NOT use a third-party service account as `tiller` for various reasons:
1. It already has some permissions we don't need
2. We need to add permissions tiller do not need
3. The permissions of tiller may change or tiller will be removed from helm leaving as with the same problem again

Using a custom ServiceAccount make sense. We should break the permissions up and double check the needed permissions (as they were never really maintained). Will add this to this pull request.

Reply:
> 2. We need to add permissions tiller do not need
Really? I don't think so but I completely agree with you that a custom ServiceAccount would be the best solution.

Reply:
> > 1. We need to add permissions tiller do not need
> 
> Really? I don't think so but I completely agree with you that a custom ServiceAccount would be the best solution.

Adding permissions to tiller was more hypothetical. Not sure, if we would need to do so now or in the future.

Reply:
Changed to `post-delete` while using a separate serviceAccount.